bindgen/callbacks.rs

```diff
@@ -231,15 +232,28 @@ pub struct DeriveInfo<'a> {
     pub kind: TypeKind,
 }
 
-/// Relevant information about a type to which new attributes will be added using
+/// Relevant information about an item to which new attributes will be added using
 /// [`ParseCallbacks::add_attributes`].
 #[derive(Debug)]
 #[non_exhaustive]
 pub struct AttributeInfo<'a> {
```

Comment:
Would it be possible to add the doc comment to this struct? 
In my case, I would like to feature guard items, based on information provided in doc comments. 
In my fork of bindgen I added a custom callback for this purpose, but perhaps I could use this callback in the future.

Reply:
You can already do this on `alias` branch:
```rust
if attrs.iter().any(|attr| attr.contains("doc")) {
   attrs.insert(r#"#[cfg(feature="guard")]"#.to_owned());
   println!("cargo:warning={attrs:#?}")
}
```
output
```shell
[phnt 0.1.2] cargo:warning={
[phnt 0.1.2]     "# [repr (C)]",
[phnt 0.1.2]     "# [derive (Debug , Copy , Clone)]",
[phnt 0.1.2]     "# [doc = \"The NTPSS_MEMORY_BULK_INFORMATION structure is used to query basic memory information in bulk for a process.\"]",
[phnt 0.1.2]     "#[cfg(feature=\"guard\")]",
[phnt 0.1.2] }
```

Reply:
Resolved as this will be functional in the final implementation. Complete <k,v> parsing of attributes is not planned as of now.